### DISCUSSION TITLE
Unable to update Sonarr

### DISCUSSION THREAD

**Discussion by sambarlick** [tteck/Proxmox#2028] [Help Needed] [answered]
Hi all,

Currently I'm running version 4.0.0.697 —11 Oct 2023 and Sonarr says I'm a few versions behind and to run apt to update. When I run apt update the result is that everything is up-to-date. I changed the source.list from testing-main main to testing-devolop main and still no updates.

It's there another source I should be pointing to?

Thanks
Sam

**Comment by tteck** [comment, marked as answer]
Depending on the creation date, try typing `update` in the LXC console.

**Reply by sambarlick** [reply]
Wow thanks. So the script must be looking at a different source the the apt update source list. Do you know if I can include that source in the apt update source list?

**Reply by tteck** [reply]
v3 updates via `apt`, v4 does not.

**Reply by sambarlick** [reply]
Ok thanks @tteck

**Comment by Vetsus** [comment]
i'm sitting on version 3.0.10.1567

when i run update it asks "Supper/Update functions for Sonarr LXC.  Proceed?  i answer 'yes' but then goes back to command line and displays an error "No Sonarr Installation Found!"

apt update seems to run ok.  apt upgrade displays "0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded."

but i'm still stuck on the same version of 3.0.10.1567

how do i get around that as it says the latest version currently is 4.0.8.1874

**Reply by che1404** [reply]
Hi! did you find a solution? I'm having the same issue.
